CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 - Add `http.request.method` attribute for performance span data ([#786](https://github.com/getsentry/sentry-go/pull/786))
 - Automatic transactions for Fasthttp integration ([#732](https://github.com/getsentry/sentry-go/pull/723))
 - Add `Fiber` integration ([#795](https://github.com/getsentry/sentry-go/pull/795))
+- Add support for errors.Unwrap() when unwrapping exceptions and new exception mechanism fields from [RFC-0079](https://github.com/getsentry/rfcs/blob/main/text/0079-exception-groups.md#new-mechanism-fields) ([#792](https://github.com/getsentry/sentry-go/pull/792))
```

Comment:
"new exception mechanism fields" is an implementation detail IMO. How about simplifying this a little, maybe:
```suggestion
- Use `errors.Unwrap()` to create exception groups ([#792](https://github.com/getsentry/sentry-go/pull/792))
```